resources/views/vendor/backpack/base/inc/sidebar_content.blade.php

```diff
@@ -1,14 +1,22 @@
-<li class='nav-item'><a class='nav-link' href='{{ backpack_url('item') }}'><i class="las la-dice-d20"></i> Items</a></li>
-<li class='nav-item'><a class='nav-link' href='{{ backpack_url('league') }}'><i class="las la-users"></i> Leagues</a></li>
-<li class='nav-item'><a class='nav-link' href='{{ backpack_url('rating') }}'><i class="las la-chart-bar"></i> Ratings</a></li>
-<li class='nav-item'><a class='nav-link' href='{{ backpack_url('role') }}'><i class="las la-user-tag"></i> Roles</a></li>
-<li class='nav-item'><a class='nav-link' href='{{ backpack_url('session') }}'><i class="las la-chair"></i> Sessions</a></li>
-<li class='nav-item'><a class='nav-link' href='{{ backpack_url('trade') }}'><i class="las la-exchange-alt"></i> Trades</a></li>
-<li class='nav-item'><a class='nav-link' href='{{ backpack_url('user') }}'><i class="las la-user"></i> Users</a></li>
+@if (Auth::user()->isSiteAdmin())
+    <li class="nav-item"><a class="nav-link" href="{{ backpack_url('dashboard') }}"><i class="la la-dice-d20 nav-icon"></i> {{ trans('backpack::base.dashboard') }}</a></li>
+    <li class='nav-item'><a class='nav-link' href='{{ backpack_url('adventure') }}'><i class="las la-dragon"></i> Adventures</a></li>
+    <li class='nav-item'><a class='nav-link' href='{{ backpack_url('campaign') }}'><i class="las la-user-friends"></i> Campaigns</a></li>
+    <li class='nav-item'><a class='nav-link' href='{{ backpack_url('character') }}'><i class="las la-user-shield"></i> Characters</a></li>
+    <li class='nav-item'><a class='nav-link' href='{{ backpack_url('entry') }}'><i class="las la-pen-fancy"></i> Entries</a></li>
+    <li class='nav-item'><a class='nav-link' href='{{ backpack_url('event') }}'><i class="las la-calendar"></i> Events</a></li>
+    <li class='nav-item'><a class='nav-link' href='{{ backpack_url('item') }}'><i class="las la-dice-d20"></i> Items</a></li>
+    <li class='nav-item'><a class='nav-link' href='{{ backpack_url('league') }}'><i class="las la-users"></i> Leagues</a></li>
+    <li class='nav-item'><a class='nav-link' href='{{ backpack_url('rating') }}'><i class="las la-chart-bar"></i> Ratings</a></li>
+    <li class='nav-item'><a class='nav-link' href='{{ backpack_url('role') }}'><i class="las la-user-tag"></i> Roles</a></li>
+    <li class='nav-item'><a class='nav-link' href='{{ backpack_url('session') }}'><i class="las la-chair"></i> Sessions</a></li>
+    <li class='nav-item'><a class='nav-link' href='{{ backpack_url('trade') }}'><i class="las la-exchange-alt"></i> Trades</a></li>
+    <li class='nav-item'><a class='nav-link' href='{{ backpack_url('user') }}'><i class="las la-user"></i> Users</a></li>
+@elseif (Auth::user()->isLeagueAdminRole())
+    <li class='nav-item'><a class='nav-link' href='{{ backpack_url('event') }}'><i class="las la-calendar"></i> Events</a></li>
+    <li class='nav-item'><a class='nav-link' href='{{ backpack_url('session') }}'><i class="las la-chair"></i> Sessions</a></li>
+@endif
```

Comment:
The admins do see the event and session links because they are also included in the if statement. If I were to implement your suggestion the event and session will no longer be in alphabetical order for admins and will appear at the bottom of the sidebar. It would work but would be weird in my opinion to not be in alphabetical order.